### PR TITLE
Fix variable type

### PR DESCRIPTION
**Pull request by krekos**
Fix compatibility with Nette/Image

### CODE REVIEW

**Comment by krekos** [comment]
FIX of the same thing but on the side of Nette/images.

https://github.com/nette/utils/pull/217

File: src/Storages/LocalStorage.php [hunk 1]
```diff
@@ -170,6 +170,8 @@ protected function getLink(IResource $resource): ?string {
 				$image = $this->imageFactory->createFromFile($originalPath);
 			} catch (UnknownImageFileException $e) {
 				return null;
+			} catch (ImageException $e){
```
Comment (MartkCz) [inline src/Storages/LocalStorage.php]: `$e){` to `$e) {`

File: src/Modifiers/BaseModifiers.php [hunk 2]
```diff
@@ -104,6 +108,11 @@ public function resize(ModifierParam $param, $width, $height, $flag = Image::FIT
 		if ($flag) {
 			$flag = $this->converseFlags(array_slice(func_get_args(), 3));
 		}
+		
+		// if is variable string, but '%' sign is missing, then cast it to int
+		$width = is_string($width) && substr($width, -1) !== '%' ? (int)$width : $width;
+		$height = is_string($height) && substr($height, -1) !== '%' ? (int)$height : $height;
```
Comment (MartkCz) [inline src/Modifiers/BaseModifiers.php]: `(int)$var` to `(int) $var`

File: src/Modifiers/BaseModifiers.php [hunk 3]
```diff
@@ -26,6 +26,10 @@ public function load(IModifiers $modifiers): void {
 		$modifiers->addModifier('crop', function (ModifierParam $param, $left, $top, $width, $height) {
 			$image = $param->getImage();
 
+			// if is variable string, but '%' sign is missing, then cast it to int
+			$width = is_string($width) && substr($width, -1) !== '%' ? (int)$width : $width;
+			$height = is_string($height) && substr($height, -1) !== '%' ? (int)$height : $height;
```
Comment (MartkCz) [inline src/Modifiers/BaseModifiers.php]: `(int)$var` to `(int) $var`

**Review by MartkCz — CHANGES_REQUESTED** [review]

**Pull request merged**